WordPress/src/main/java/org/wordpress/android/ui/prefs/AccountSettingsFragment.java

```diff
@@ -328,6 +337,50 @@ public void onAccountChanged(OnAccountChanged event) {
         }
     }
 
+    /**
+     * If the username can be changed then the control can be clicked to open to the
+     * Username Changer screen.
+     */
+    private void checkIfUsernameCanBeChanged() {
+        AccountModel account = mAccountStore.getAccount();
+        mUsernamePreference.setEnabled(account.getUsernameCanBeChanged());
+        mUsernamePreference.setOnPreferenceClickListener(preference -> {
+            showUsernameChangerFragment();
+            return false;
```

Comment:
I'm wondering if we should return "true" here. Wdyt?

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10436/commits/01c8d819703b7a848b1c702ad46b98e59d03ce9e